MapboxNavigation/RouteVoiceController.swift

```diff
@@ -181,16 +185,16 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate, AVAudioP
         return true
     }
     
-    func speak(_ text: String, error: String? = nil) {
+    func speak(_ text: String, error: Error?) {
         // Note why it failed
         if let error = error {
-            print(error)
+            voiceControllerDelegate?.voiceController?(self, spokenInstructionsDidFailWith: error)
```

Comment:
It should be the responsibility of the caller of this method to deal with any error that is being passed in. This method should only be responsible for handling errors that arise in the execution of this method.

/ref https://github.com/mapbox/mapbox-navigation-ios/pull/800#discussion_r149505637